langchain4j-core/src/test/java/dev/langchain4j/store/embedding/EmbeddingStoreIT.java

```diff
@@ -34,6 +34,7 @@ void should_add_embedding_with_segment_with_metadata() {
         }
 
         awaitUntilPersisted();
+        awaitUntilPersisted(embedding, 1);
```

Comment:
Needed to add this function to be able to use Awaitility with pinecone client to wait for records to be retrievable.
Pinecone is eventually consistent, so records are/may not be available just after write